ArduCopter/mode.h

```diff
@@ -835,6 +835,14 @@ class ModeFlip : public Mode {
     using Mode::Mode;
     Number mode_number() const override { return Number::FLIP; }
 
+// FLIP_MODE parameters
+    enum class Dir{
```

Comment:
I think this could be private or protected?
```suggestion
    enum class Dir {
```

Reply:
I made it public so it could be available in Parameters.h.

I can make it private if the team is alright with the default being an integer in Parameters.h.

Reply:
Your right, better to use the enum for the default.